pyxrf/db_config/hxn_db_config.py

```diff
@@ -0,0 +1,71 @@
+from metadatastore.mds import MDS
+from databroker import Broker
+from databroker.core import register_builtin_handlers
+from filestore.fs import FileStore
+
+_mds_config = {'host': 'xf03id-ca1',
```

Comment:
I think it is a little surprising to find HXN-specific configuration shipped with a tool that is nominally general-purpose. Is that another way to inject the location of a configuration file?

Reply:
I am going to update this to conform with new databroker. So this information can be deleted. Thanks for checking this.